CHANGELOG.md

```diff
@@ -6,11 +6,16 @@ IMPROVEMENTS:
 * CRDs: update the CRD versions from v1beta1 to v1. [[GH-883](https://github.com/hashicorp/consul-helm/pull/883)]
 * Enterprise: support applying Consul Enterprise license when security context defaults to non-root users. [[GH-880](https://github.com/hashicorp/consul-helm/pull/880)]
 * Sync Catalog: add new `syncCatalog.extraLabels` Helm value for configuring labels on sync catalog pods. [[GH-892](https://github.com/hashicorp/consul-helm/pull/892)]
+* Connect: Support leader election for connect webhook deployment. [[GH-903](https://github.com/hashicorp/consul-helm/pull/903)]
```

Comment:
Can we say something that this now allows multiple instances/HA of the connect-inject deployment?